.gitignore

```diff
@@ -24,3 +24,4 @@ yarn.lock
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+.history
```

Comment:
```suggestion
.history/*
```

Reply:
I found that adding the .history directory to the .gitignore file doesn't change much since git was already tracking the directory when I made my first push. @akhenda made a recommendation I use `git rm --cached -r .history` it deleted all the history files.